lib/jido_signal.ex

```diff
@@ -148,9 +148,18 @@ defmodule Jido.Signal do
   import Jido.Signal.Ext.Registry, only: [get: 1]
 
   alias Jido.Signal.Dispatch
+  alias Jido.Signal.Ext
   alias Jido.Signal.ID
+  alias Jido.Signal.Serialization.Serializer
   alias Jido.Signal.Serialization.TypeProvider
 
+  require Logger
+
+  @core_attrs ~w[
+    specversion id source type subject time
+    datacontenttype dataschema data jido_dispatch extensions
+  ]a
```

Comment:
I didn't know we could use brackets 
I usually do something like this:
```elixir
@core_attrs ~w(
  specversion
  id
  source
  type
  subject
  time
  datacontenttype
  dataschema
  data
  jido_dispatch
  extensions
 )a
 ```
 definitely not a huge deal